### PR TITLE
Change docs-searchbar.js description to deprecated

### PR DESCRIPTION
**Pull request by PzaThief**
# Pull Request

## What does this PR do?
- Change docs-searchbar.js description to deprecated following docs-searchbar.js repository

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: learn/what_is_meilisearch/sdks.mdx [hunk 1]
```diff
@@ -37,7 +37,7 @@ You can use Meilisearch API wrappers in your favorite language. These libraries
 - [Vue](https://github.com/meilisearch/meilisearch-vue)
 - [Instant Meilisearch](https://github.com/meilisearch/meilisearch-js-plugins/tree/main/packages/instant-meilisearch)
 - [Autocomplete client](https://github.com/meilisearch/meilisearch-js-plugins/tree/main/packages/autocomplete-client)
-- [docs-searchbar.js](https://github.com/meilisearch/docs-searchbar.js): a search bar integration for all kinds of documentation.
+- [docs-searchbar.js](https://github.com/meilisearch/docs-searchbar.js) (deprecated)
```
Comment (guimachiavelli) [inline learn/what_is_meilisearch/sdks.mdx:40]:
```suggestion
- [docs-searchbar.js](https://github.com/tauri-apps/meilisearch-docsearch)
```

**Review by guimachiavelli — CHANGES_REQUESTED** [review]
Hello @PzaThief, and thank you for taking the time to contribute to our repo.

I have discussed the deprecated docs-searchbar repo with the rest of team and we think it would be more useful to direct users to a maintained fork. This way they can continue moving forward with their research.

I made a suggestion on the PR. If you agree with our decision, please commit it and I'll merge this PR ASAP.

Thanks again!

**Comment by PzaThief** [comment]
> Hello @PzaThief, and thank you for taking the time to contribute to our repo.
> 
> I have discussed the deprecated docs-searchbar repo with the rest of team and we think it would be more useful to direct users to a maintained fork. This way they can continue moving forward with their research.
> 
> I made a suggestion on the PR. If you agree with our decision, please commit it and I'll merge this PR ASAP.
> 
> Thanks again!

Thanks for your suggestion. I agree with your opinion so committed it.

**Review by guimachiavelli — APPROVED** [review]
Looks good to me! Thanks again, @PzaThief! ☺️

**Pull request merged**